BacktraceScreen.h

```diff
@@ -0,0 +1,72 @@
+#ifndef HEADER_BacktraceScreen
+#define HEADER_BacktraceScreen
+/*
+htop - BacktraceScreen.h
+(C) 2023 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include <stddef.h>
+#include <stdbool.h>
+
+#include "Object.h"
+#include "Panel.h"
+#include "Process.h"
+#include "Settings.h"
+
+typedef struct BacktracePanelPrintingHelper_ {
+   size_t maxAddressLength;
+   size_t maxDemangledFunctionNameLength;
+   size_t maxFunctionNameLength;
+   size_t maxNumberFrameLength;
+   size_t maxObjectPathLength;
+   size_t maxObjectNameLength;
+} BacktracePanelPrintingHelper;
```

Comment:
I never know how to name my variable :smile:. I changed it for `maxFrameNumberLength`.
I propose : 
```c
size_t maxAddrLen;
size_t maxDemangledFuncNameLen;
size_t maxFuncNameLen;
size_t maxFrameNumLen;
size_t maxObjPathLen;
size_t maxObjNameLen;
```
Is it better?

Reply:
Since you have many length values to calculate, how about you name your structure `BacktracePanelLengthValues` and you can avoid having the `Len` name in all of your members?

Reply:
When I created this structure, I didn't know what I would put inside. And I chose a generic name. This is why it is called `BacktracePanelPrintingHelper`. It suits me well because it helps to print the frame. 
Moreover, in the future, it will be able to help the dev to print some information without creating a new structure.

So, I prefer to keep the actual name even if it's longer or `Len` can be removed.